validation/createCompanySchema.js

```diff
@@ -0,0 +1,26 @@
+import Joi from '@hapi/joi';
+
+export const createCompanySchema = Joi.object({
+  companyName: Joi.string()
+    .alphanum()
+    .min(3)
+    .max(30)
+    .required(),
+
+  acceptedTerms: Joi.bool().required(),
+  copyAcceptedTerms: Joi.bool().required(),
+  email: Joi.string()
+    .email()
+    .required(),
+  contactFirstName: Joi.string().required(),
+  companyLastName: Joi.string().required(),
+  iban: Joi.string().required(),
+  address: Joi.object({
+    city: Joi.string().required(),
+    houseNumber: Joi.string().required(),
+    street: Joi.string().required(),
+    zipCode: Joi.string().required(),
+    latitude: Joi.number().required(),
+    longitude: Joi.number().required()
+  })
```

Comment:
I can not find a field for the KvK number

Reply:
I was looking at the .NET project, and it does not have this property as well. I will add it here

Reply:
Funny enough, but Joi doesn't have a good solution for validating a number length of 8 :(